fees.js

```diff
@@ -41,10 +42,10 @@ async function get_min_next_block_fee_rate() {
     }
     if (MIN_FEE_BUFFER_PERCENT) {
         const min_fee_buffer_percent = parseFloat(process.env.MIN_FEE_BUFFER_PERCENT)
-        return Math.round((data[0].feeRange[0] * min_fee_buffer_percent) * 10) / 10
+        return Math.ceil((data[0].feeRange[NEXT_BLOCK_FEE_SLOT] * min_fee_buffer_percent) * 10) / 10
```

Comment:
Shall we cover this method `get_fee_rate` with unit tests?
changing round to ceil should cause unit test failure as this is a change in logic